### DISCUSSION TITLE
generate a helm-chart automatically

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#1582] [Ideas] [answered]
WIP: https://github.com/nextcloud/all-in-one/pull/1574

follow-ups:
- [x]  https://github.com/nextcloud/all-in-one/pull/1848
- [x] https://github.com/nextcloud/all-in-one/pull/1912
- [x] https://github.com/nextcloud/all-in-one/pull/1934
- [x] test if it really works 
```
helm upgrade \
    --install \
    --create-namespace \
    -n trilium \
    -f ~/some/path/trilium-notes.yaml \
    trilium \
    ~/Code/charts/charts/trilium-notes/
```
In dem Verzeichnis `~/Code/charts/charts/trilium-notes/` sind dann wie gewohnt die Dateien `values.yaml` und `Chart.yaml` und der templates Ordner.
- [x] implement releases https://github.com/nextcloud/all-in-one/pull/2034
(https://github.com/helm/chart-releaser/issues/122 and https://github.com/dunglas/mercure/pull/515 together with [make-release-latest](https://github.com/helm/chart-releaser/blob/ee26f50ce93ca23920dc1ce93e677665ce65a46e/README.md?plain=1#L88) and https://medium.com/@stefanprodan/automate-helm-chart-repository-publishing-with-github-actions-and-pages-8a374ce24cf4 and https://github.com/nextcloud/helm/blob/master/.github/workflows/release.yaml)

**Comment by ghost** [comment]
That's great!  if i can help in any way please contact me :)

**Comment by ghost** [comment]
is an inclusion in the documentation planned?

**Reply by szaimen** [reply]
The helm chart is not ready yet. I will let you know when that is the case. Documentation will be available then as well.

**Reply by ghost** [reply]
Is there already a date until the Helm Chart is ready?

**Reply by szaimen** [reply]
It mostly depends on final testing and https://github.com/nextcloud/all-in-one/pull/2034 which needs https://github.com/helm/chart-releaser-action/pull/135

**Comment by szaimen** [comment, marked as answer]
Remaining pieces in https://github.com/nextcloud/all-in-one/issues/2366 and https://github.com/nextcloud/all-in-one/issues/2370
